src/Symfony/Component/Runtime/composer.json

```diff
@@ -21,7 +21,7 @@
     },
     "require-dev": {
         "composer/composer": "^1.0.2|^2.0",
-        "symfony/console": "^4.4|^5.0|^6.0",
```

Comment:
@nicolas-grekas would you agree with this change? (it removes quite some versions from the test)

There is a `.phpt` test that must check different outputs for <5.4 and >=5.4. I don't know of any way to conditionally run `.phpt` tests based on class existence (in PHPunit, I would normally create 2 tests and use `markTestAsSkipped()`).

Reply:
I'd rather relax the test case on 5.4